projects/mupdf/pdf_fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+close_fd_mask = 3
```

Comment:
That works, but would it be possible to disable noisy logging completely? Maybe there is a compilation flag, or some API like `set_log_level()` which we can use.

Reply:
If there is, I was unable to find it. Based on the code it looks like errors are always printed to stderr.